runtime/js/99_main.js

```diff
@@ -339,6 +351,7 @@ delete Object.prototype.__proto__;
     URL: util.nonEnumerable(url.URL),
     URLSearchParams: util.nonEnumerable(url.URLSearchParams),
     WebSocket: util.nonEnumerable(webSocket.WebSocket),
+    WebSocketStream: util.nonEnumerable(webSocket.WebSocketStream),
```

Comment:
Can you change this to only be registered when `--unstable` is passed? I think `typeof WebSocketStream  === "undefined"` should be true when user does not pass `--unstable`.